packages/grid/x-data-grid/src/components/panel/GridPanel.tsx

```diff
@@ -27,7 +27,10 @@ export interface GridPanelProps
   open: boolean;
 }
 
-export const gridPanelClasses = generateUtilityClasses('MuiDataGrid', ['panel', 'paper']);
+export const gridPanelClasses = generateUtilityClasses<keyof GridPanelClasses>('MuiDataGrid', [
```

Comment:
Adding the generic to make sure that TS do not accept adding a class here but not on the interface